test/e2e/suite_test.go

```diff
@@ -943,6 +947,45 @@ func createMultiDatacenterCluster(t *testing.T, ctx context.Context, namespace s
 	assert.NoError(t, err, "timed out waiting for nodetool status check against "+pod)
 }
 
+// createMultiDatacenterDseCluster creates a K8ssandraCluster with two CassandraDatacenters,
+// one running locally and the other running in a remote cluster.
+// Each CassandraDatacenter is configured with a different DSE workload, with the second DC being
+// processed first due to the priority of its workload.
+func createMultiDatacenterDseCluster(t *testing.T, ctx context.Context, namespace string, f *framework.E2eFramework) {
```

Comment:
Don't you think we should move this test to dse_test.go? This file is already very large :-)

Reply:
oh yeah, totally right.
I'll move it there.